### ISSUE TITLE
Ubuntu instructions are incorrect

### ISSUE THREAD

**Issue by richlander** [dotnet/docs#33539]
In reference to: https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu

There are a couple of problems we should consider resolving:

- The docs are hard to follow w/rt Ubuntu feeds vs PMC. I think separate docs would be better. Navigating the various Ubuntu versions is hard enough.
- The docs recommend mention that .NET 6 is available via Ubuntu feeds and then have a large box to say that .NET 7 isn't available (see below). That seems like overkill and could be handled in the same sentence that talks about .NET 6.
- The PMC sections make no mention of how to install the .NET 6 SDK. I think we should document all the in-support package names (for the SDK and runtimes). For folks that are solely using .NET 6, installing the .NET 6 SDK seems easier (no need to install runtimes separately).
- If you follow the PMC instructions, you'll get .NET 7 from PMC and .NET 6 from Ubuntu. We need to add `Pin-Priority` information from scenario 2 from https://github.com/dotnet/core/issues/7699. By default, if you use PMC, you should use it in its entirety.

<img width="706" alt="image" src="https://user-images.githubusercontent.com/2608468/212439803-96770cbe-f415-4775-a048-15d48b7e79d0.png">

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 80707602-f283-e796-d3b8-3287fbc011dc
* Version Independent ID: 1d732b15-9e64-633a-000c-ad96d9d3818a
* Content: [Install .NET on Ubuntu - .NET](https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu)
* Content Source: [docs/core/install/linux-ubuntu.md](https://github.com/dotnet/docs/blob/main/docs/core/install/linux-ubuntu.md)
* Product: **.net**
* Technology: **dotnet-installation**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

---
[Associated WorkItem - 62047](https://dev.azure.com/msft-skilling/Content/_workitems/edit/62047)

**Comment by Swimburger** [comment]
I was [trying to install .NET 7 and 6 following those instructions](url), and ended up installing 7 from PMC and 6 from Ubuntu.
I didn't realize this was gonna cause issues, but it resulted in the SDK being detected, but not the runtime and I couldn't run .NET 6 apps. [Details here](https://gist.github.com/Swimburger/eec9e26f029875c2a196f74549c29388).

After uninstalling both and following the instruction from [this issue: Clean machine scenario 2](https://github.com/dotnet/core/issues/7699), then installing .NET 7, I was able to resolve the issue, installing both from PMC.
